src/ext/graphql/transmodelapi/schema.graphql

```diff
@@ -295,6 +303,8 @@ type Leg {
   distance: Float!
   "The leg's duration in seconds"
   duration: Long!
+  "The leg's elevation profile."
```

Comment:
I think the comment should define what to expect here: 
- what is the elevation at the start coordinate - 0m? 
- is negative values allowed?
- is the distance dx uniform? 
- is start and end place/coordinate included?

Reply:
I updated the documentation with all of the points that you requested apart from the dx. Looking at the response I think you get a sample every 10meter-ish but I'm not completely sure. Do you know?